rpc/data_transfer.go

```diff
@@ -180,6 +180,15 @@ func (api *DataTransferAPI) SendDataQueryRequest(r *http.Request, args *SendData
 		return fmt.Errorf("failed to insert data query request: %w", err)
 	}
 
+	//Purge QueryHistory after 5 minutes
+	go func() {
```

Comment:
This functionality will be moved out to main.go and using a time ticker of every 30 seconds it will call `PurgeQueryHistory`

Reply:
@filefilegoadmin done